plugins/modules/aci_rest.py

```diff
@@ -537,5 +546,25 @@ def main():
     aci.exit_json(**aci.result)
 
 
+def process_rest_payload(payload, convert_values_to_str):
+    if convert_values_to_str:
+        if isinstance(payload, str):
+            payload = convert_payload_values_to_str(json.loads(payload))
+        else:
+            payload = convert_payload_values_to_str(payload)
+    return payload
+
+
+def convert_payload_values_to_str(data):
+    if isinstance(data, dict):
+        return {k: convert_payload_values_to_str(v) for k, v in data.items()}
+    elif isinstance(data, list):
+        return [convert_payload_values_to_str(item) for item in data]
+    elif isinstance(data, int) or isinstance(data, bool) or isinstance(data, float):
```

Comment:
Does this handle JSON Null types? Looks like it does as `null` in JSON won't be converted to a string. 
Would it be possible to add a test when a JSON value is explicitly set to `null`?

Reply:
Done!